src/components/LineChartSmallMult/LineChartSmallMult.jsx

```diff
@@ -41,6 +43,19 @@ export default class LineChartSmallMult extends PureComponent {
     metrics: [],
   }
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hover: false,
+      mouse: { x: 0, y: 0 },
+    };
```

Comment:
using state! what a life. but I think it's reasonable for performance purposes and I don't foresee and external highlighting impacting the chart/the highlighting in the small mults being used externally.